src/client/user/components/CreateUrlModal/ModalMargins.tsx

```diff
@@ -1,25 +1,33 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, { FunctionComponent } from 'react'
 import { createStyles, makeStyles } from '@material-ui/core'
-
 import { ApplyAppMargins } from '../../../app/components/AppMargins'
 import useFullScreenDialog from '../../helpers/fullScreenDialog'
 
+type StyleProps = {
+  applyLeftMargin?: boolean,
+  applyRightMargin?: boolean,
```

Comment:
we should remove `applyLeftMargin` and `applyRightMargin` from the codebase if they're never actually used or overridden.

Reply:
applyRightMargin is used in CreateUrlModal, but applyLeftMargin is not used at all. Will remove applyLeftMargin